packages/perseus-core/src/utils/split-perseus-renderer.ts

```diff
@@ -22,13 +22,24 @@ export default function splitPerseusRenderer(
     const splitWidgets = {};
 
     for (const [id, widget] of Object.entries(upgradedWidgets)) {
-        const publicWidgetOptionsFun = getPublicWidgetOptionsFunction(
-            widget.type,
-        );
-        splitWidgets[id] = {
-            ...widget,
-            options: publicWidgetOptionsFun(widget.options as any),
-        };
+        /**
+         * Static widgets are presentational, used to represent concepts
+         * rather than collect answers from learners. For instance we might
+         * want a static InteractiveGraph that shows learners a chart before
+         * we give them a Radio widget to check their understanding of the chart.
+         * We need to keep the answers in this case because it's pre-scoring information.
+         */
+        if (widget.static) {
+            splitWidgets[id] = widget;
+        } else {
```

Comment:
I'm curious, were you able to see an issue in Storybook or in some version of prod that this fixes? I'm all for it, but I'd love to know if it is actually working via a manual test. It seems to have the same outcome when I tested in prod storybook versus local storybook for this branch, so wondering how to manual test to see it work 🤔

Reply:
I think in this case since it's operating on data (not a UI change) it should be enough to assert the data is in the right shape.